### ISSUE TITLE
Cannot unlock drive with new gfx: RX 5500 XT

### ISSUE THREAD

**Issue by NQbbe** [Drive-Trust-Alliance/sedutil#327]
I know this sounds absolutely nuts, but after doing nothing but replacing my RX 570 with an RX 5500 XT I can no longer unlock my drive.

After entering the password it simply says "Is OPAL Failed"
I've noticed the keyboard input is extremely laggy when entering the password (using latest UEFI PBA image), so much so that I had to slow down typing to ensure each letter is actually entered. This never happened with the old card, so I'm guessing something with the new card breaks my keyboard entries, and as such it won't unlock.

2 fun parts:
1. It works fine with an RX 5600 XT (which is based on the older RX 5700, so maybe it has better UEFI support?)
2. The samme issue happens if I keep my RX 570 but replace my Ryzen 1800X with a Ryzen 3600.

This leads me to believe the issue may have something to do with my somewhat old BIOS. I've tried these hardware combinations on both an B350 and X370 board, each updated to latest BIOS which *should* support Ryzen 3000 series, albeit it's labeled as a beta by the manufacturer (MSI) and it's from late 2019.

Long story short: Do you think this is an issue in the PBA image or simply my motherboards being too old?

And is there anything I can do to debug why this is happening?

Thanks in advance :-)

**Comment by NQbbe** [comment]
I searched a bit in the open issues and found a solution: ChubbyAnt's fork with improved Ryzen support worked like a charm.
I'm actually running both the Ryzen 3600 and the RX 5500 XT now and it still works :-)

Here's a link to the fork which solved it: https://github.com/ChubbyAnt/sedutil
